redhat-operators/kie-cloud-operator/kiecloud-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,210 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: kiecloud-operator.v0.1.0
+  namespace: placeholder
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
```

Comment:
This should be a comma separated list. Maybe try: `pam`

Reply:
fixed